docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+
+   Administration console access is available at http://wlswls1596-wlsresourcegrou-685ba0-7434b4f5.hcp.eastus.azmk8s.io:30701/console
+   The following files were generated:
+     /home/username/azure/weblogic-domains/domain1/create-domain-inputs.yaml
+     /home/username/azure/weblogic-domains/domain1/create-domain-job.yaml
+     /home/username/azure/weblogic-domains/domain1/domain.yaml
+
+   Completed
+   ```
+   {{% /expand %}}
+
+   > **Note**: If your output does not show a successful completion, you must
+   troubleshoot the reason and resolve it before proceeding to the next
+   step.
+
+4. You must create `LoadBalancer` services for the Administration Server and the WLS cluster.  This enables WLS to service requests from outside the AKS cluster.
+
+   Use the sample configuration file `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/domain-on-pv/admin-lb.yaml` to create a load balancer service for the Administration Server. If you are choosing not to use the predefined YAML file and instead created new one with customized values, then substitute the following content with your domain values.
+
+   ```yaml
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: domain1-admin-server-external-lb
+     namespace: default
+   spec:
+     ports:
+     - name: default
+       port: 7001
+       protocol: TCP
+       targetPort: 7001
+     selector:
+       weblogic.domainUID: domain1
+       weblogic.serverName: admin-server
+     sessionAffinity: None
+     type: LoadBalancer
+   ```
+
+   Use the sample configuration file `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/domain-on-pv/cluster-lb.yaml` to create a load balancer service for the managed servers. If you are choosing not to use the predefined YAML file and instead created new one with customized values, then substitute the following content with you domain values.
+
+   ```yaml
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: domain1-cluster-1-lb
+     namespace: default
+   spec:
+     ports:
+     - name: default
+       port: 8001
+       protocol: TCP
+       targetPort: 8001
+     selector:
+       weblogic.domainUID: domain1
+       weblogic.clusterName: cluster-1
+     sessionAffinity: None
+     type: LoadBalancer
+   ```
+
+   Create the load balancer services using the following commands:
+
+   ```bash
+   $ kubectl apply -f ~/azure/weblogic-on-aks/admin-lb.yaml
+   service/domain1-admin-server-external-lb created
+
+   $ kubectl  apply -f ~/azure/weblogic-on-aks/cluster-lb.yaml
+   service/domain1-cluster-1-external-lb created
+   ```
+
+   After a short time, you will see the Administration Server and Managed Servers running.
+
+   Use the following command to check server pod status:
+
+   ```bash
+   $ kubectl get pods --watch
+   ```
+
+   It may take you up to 20 minutes to deploy all pods, please wait and make sure everything is ready.
```

Comment:
It may take you up to -> It may take up to
all pods, please -> all pods; please (semi-colon, two sentences)